src/store/index.js

```diff
@@ -38,11 +48,36 @@ export default new Vuex.Store({
 		isBackdropActive(state) {
 			return state.backdrop;
 		},
+		isMessageActive(state) {
+			return state.message;
+		},
+		textMessage(state) {
+			return state.message_text;
+		},
 	},
 	actions: {
 		setLoading(context, status) {
```

Comment:
what about renaming this to `payload`, as its going to be object in some cases, and `status` would mean some primitive, not object.